buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -318,7 +305,7 @@ class GuideAsciidocGenerator {
 
         List<String> newLines = commonLines(file, projectDir)
 
-        String pattern = "(\\{(\\d+)(:?([UL])?)})"
+        String pattern = "(\\{(\\d+)(_?([UL])?)})"
```

Comment:
why are we changing the pattern?

Reply:
necessary because otherwise the old parser would not do the substitution since the syntax changed